### ISSUE TITLE
motion cancellation noise

### ISSUE THREAD

**Issue by iamisti** [matzman666/OpenVR-InputEmulator#146]
Hi guys,

I've bought a vive tracker and set up everything using this piece of software as well.
My problem is, when i turn on motion canellation, i got quite a noticable noise. the whole picture is shaking a little bit and it makes me very sick. I've tried all the options, nothing worked.
Does this motion cancellation for you guys works without any noise? what is the solution to smooth it, or even remove the noise?

**Comment by trueA1M2** [comment]
this is maybe the same as bridge collapsing due to frequency, I have sub bass shakers 500watt in total I have to cut the hz at 80-90 or everything goes mad

**Comment by iamisti** [comment]
thanks man forhelping, really appreciate it!
So your setup is smooth as hell, no noise at all?
Also can you give me any suggestion why my tracker is not showing up in openvr? i see 1 headset and 2 controllers (only one controller turned on tho) and i cant see my vive tracker in the list

**Comment by trueA1M2** [comment]
Im not sure how many things get signed in but they need unplugging before you turn on steam

**Comment by iamisti** [comment]
okay but pls tell me if your setup is working completely smooth.
So you cant notice wny shuttering or noise. Is that right?

**Comment by trueA1M2** [comment]
No stuttering it sounds like something else don't use steam beta least it works ok try deleting openvrpaths as is in comments 143

**Comment by iamisti** [comment]
yeah i dod that. Im not using beta steam. either.
Btw how is your nase stations aligned? you have one on your ceiling as in the tutorial? i only have on the left and on the right next to me. Could that be the issue?

**Comment by trueA1M2** [comment]
As 143 show my htc vive controller I have one scanner in front and one behind in the corners of the room approx. 3.5m apart I sit approx. 1.5m from the forward one, this is my same game area for everything. Before the sub bass shakers my motion comp was 100% every time now I get an odd skip/wander with things tuned out

**Comment by iamisti** [comment]
despite all my effort the shuttering is still there. When i turn motion cancellation off, then everything is super smooth, but as soon as it's on, no matter what setup i use (disable, set zero, etc..) there is a significant shuttering by default, and it increases if i cornering and breaking. Please someone tell me what's wrong with my setup. It has to be a software issue, because my headset is tracked extremely smooth by default.

**Comment by trueA1M2** [comment]
I don't know what else to suggest, check controllers aren't being used in steam home ie key mapping or maybe try a clean reinstall make sure you wipe out all old user and program folders

**Comment by trueA1M2** [comment]
have you lowered the graphics to their lowest settings and turned off or reduced super sampling in steamvr settings

**Comment by iamisti** [comment]
yes i did. Lowest possible. Same result. So it should not be performance bottleneck. I have a very strong machine, 1080ti 11gb, 32gb ram, i7 cpu

**Comment by wayne5w** [comment]
I am having the same issue. I have a tracker mounted on top of my seat. When I selection motion in openVR I get a ton of shaking. Turn it off and the view is rock solid.

**Comment by wayne5w** [comment]
I think it's vibration that is causing the problem. I have 4 mini LFE with simvibe and as a test I turned that off and it is 90% better. I can't go without simvibe so need to find a solution.

**Comment by trueA1M2** [comment]
I have 500 watt sub base shakers I had to drop sim vibe and clone my sound cards so I used the second graphic to tune out this its around 80hz plus, sim vibe sucks compared to a sound base and graphic

**Comment by wayne5w** [comment]
I disagree about sim vibe. I have it on a SimX stage 4+, one on each corner properly isolated and it's incredible. If I had to choose between Simvibe and motion I'd pick Simvibe. Both the vibration from Simvibe and the vibration from the seat mover actuators is messing with the vive tracker. Have tried to isolate it by mounting on rubber which helped but not enough.

**Comment by trueA1M2** [comment]
Sim vibe whilst it is a super thing i would agree doesn't work with steamvr compositor and flicks around and then the vibrations. I couldn't live with no compensation so for vr I cloned sound cards and tuned down the vibration, i like a little bit its realistic

**Comment by trueA1M2** [comment]
![20190407_084841](https://user-images.githubusercontent.com/45529434/55680563-1b5f6c80-5913-11e9-9f83-cd9752fd057c.jpg)
I've been looking at this for myself so I tried this yesterday it works to cancel the noise 100% I turned the shakers up to full with no interference at all BUT after more testing could not get rid of after wobble from the chair moving I tightened up the elastic to the point the noise was worse than I started with
 
![20190407_084918](https://user-images.githubusercontent.com/45529434/55680619-d12abb00-5913-11e9-8969-090193c7d64e.jpg)
So I ended back here with acoustic foam and elastic it removes near all the noise has no after wobble at all I may improve on this with an acoustic foam sleeve

**Comment by trueA1M2** [comment]
![20190407_101757](https://user-images.githubusercontent.com/45529434/55682088-430c0000-5926-11e9-8c9d-8c77e02c0d62.jpg)
![20190407_111223](https://user-images.githubusercontent.com/45529434/55682090-47d0b400-5926-11e9-926d-8fceb76328e3.jpg)
With a graphic as well I have this working pretty much perfick

**Comment by trueA1M2** [comment]
I may give the new sib vibe another go now weeeeeeee

**Comment by iamisti** [comment]
@trueA1M2 
oh man perfect! I might try.
I have a similar setup and i think i get some vibrations. You can check it out here: https://twitch.tv/racelab
Tell me if I would do the same thing as you did, would I get less vibrations from it? Or is it normal?

**Comment by trueA1M2** [comment]
I cant see much from thread. Nice set-up though. When im driving without the foam and shakers on and hit a resonance point i leave the car completely. With the foam a graphic and volume control I can achieve a good shake and 99.9% motion canceling
